### PR TITLE
Add security policy with PGP keys

### LINKED ISSUE #1246: Security Policy
>give instructions for how to report a security vulnerability in your project by adding a security policy to the repository

https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository

### PR DESCRIPTION
**Pull request by TheBlueMatt**
Closes #1246.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report security vulnerabilities, including denial-of-service
+vulnerabilities via email to security@lightningdevkit.org.
```
Comment (tnull) [inline SECURITY.md:6]:
```suggestion
vulnerabilities, via email to security@lightningdevkit.org.
```

**Review by tnull — APPROVED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Security Policy
```
Comment (ariard) [inline SECURITY.md:1]: Any thinking how long are we going to backport security fixes ? Or we just advice users to upgrade to latest versions ?

**Review by ariard — DISMISSED** [review]
Fingerprint attached to my name is correct.

```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

commit 0a06a6691dc54b5290e2198db8526eb46ece68ea
Author: Matt Corallo <git@bluematt.me>
Date:   Sat Jul 2 15:29:50 2022 +0000

    Add security policy with PGP keys
    
    Closes #1246.

diff --git a/SECURITY.md b/SECURITY.md
new file mode 100644
index 00000000..c740a188
- --- /dev/null
+++ b/SECURITY.md
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report security vulnerabilities, including denial-of-service
+vulnerabilities via email to security@lightningdevkit.org.
+
+If possible, please PGP-encrypt such emails to the following keys (available on
+keyservers and WKD via `gpg --auto-key-locate wkd  --locate-external-keys
+ldk-security-1@mattcorallo.com ldk-security-2@mattcorallo.com`). Please include
+your own public key as an attachment or inline for replies.
+
+ * 07DF3E57A548CCFB7530709189BBB8663E2E65CE (Matt Corallo)
+ * 5DBC576CCCF546CA72AB06CE912EF12EA67705F5 (Jeffrey Czyz)
+ * 729E9D9D92C75A5FBFEEE057B5DD717BEF7CA5B1 (Wilmer Paulino)
+ * BD6EED4D339EDBF7E7CE7F8836153082BDF676FD (Elias Rohrer)
+ * 6E0287D8849AE741E47CC586FD3E106A2CE099B4 (Valentine Wallace)
+ * 69CFEA635D0E6E6F13FD9D9136D932FCAC0305F0 (Arik Sosman)
+ * A5A6868D7AA91DD00AC1A67F817FFA028EF61C94 (Antoine Riard)
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCgAdFiEEpaaGjXqpHdAKwaZ/gX/6Ao72HJQFAmLDktUACgkQgX/6Ao72
HJTYYQ//ShvXv4ChTab7q6mHMYEuU+Fz22j7J37FUvY8lf8a7bGU4DB87jyzxpYx
bQ73Q3o9jl/qlciQqvtfVNrhbuphpkmWl9m2mcjUM/b9dSFJr/b0oauA1B4N87kF
OeMef5K/iiH2bvYkki5nnYOf8FCqnVUVUKRfiZsUtgStLh7tXUaShGPwM7pNsH3b
/0jDAb/dNlnf3XipUW/YPmtqC1KUrz2fyLXg7qX5kTta5q5VyGV1D28HuGfa8dMV
1N/vOt6EkcdoMwfA4AtvY2Eu2n6KUK2/q2lG7DK9IP48fOwd4uFMFCQRLpWd0/vQ
bPP9POa9NJmZ2DhYcu7W8vvu/Ssn2xgyPYvXYEkwoiwWH4V7agRO5oapRzA8xQ95
DpIz6E8szHWaosiHG4JIVGUk06mM/oZrNrhhwJTHnsDFy3RiRBeSFQ3YTFFynvgc
VCjIcq8+rw6AazhAq35k58t6fAvOcpc8GNF0fgNE6pZuT6/OSqgh/vFDrhA4yTqm
1bm3cU4afmEuAVyK+9b7SPvjDgYHOLXm7rQ7W2fGA5HC7hhJJKwadWVNjETliUXZ
goSYuid9ko1nCg/kd5uVGUwG6mHBpXrVDsEjiGAxtCosKPkbHmJEp0m2Rp0n88/F
wIYJBCrcoaGwsdivr8L1Uf1YcGu92Y1APqHCg0LzNhcahyo1PjY=
=wc9K
-----END PGP SIGNATURE-----
```

**Review by tnull — APPROVED** [review]

File: SECURITY.md (hunk 2 above)
Comment (ariard) [reply SECURITY.md:1]: Yep, for sure.

**Review by ariard — APPROVED** [review]

**Pull request merged**